components/director/pkg/graphql/schema.graphql

```diff
@@ -533,6 +620,31 @@ type OneTimeTokenForRuntime implements OneTimeToken {
 	rawEncoded: String
 }
 
+type PackageDefinition {
+	id: ID!
+	name: String!
+	description: String
+	authRequestJSONSchema: JSONSchema
```

Comment:
to discuss `authRequestJSONSchema` --> `instanceParametersJSONSchema`

Reply:
what do you think about `APIInstanceAuthRequestJSONSchema`?

Reply:
okay, then `authRequestInputSchema` will be the final name